server/src/test/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -1693,6 +1693,7 @@ public synchronized void fullRestart(RestartCallback callback) throws Exception
         }
 
         assert nodesByRoles.values().stream().mapToInt(List::size).sum() == nodes.size();
+        callback.onAllNodesStopped();
```

Comment:
In the upstream code, this line is above the assertion line. Any reason for this change?

Reply:
👍  My fault, seems like i was on the wrong state of elasticsearch, there was no assert at all when i looked at it.